src/main/java/graphql/annotations/processor/GraphQLAnnotations.java

```diff
@@ -73,6 +76,13 @@ public GraphQLAnnotations() {
         extensionsHandler.setMethodSearchAlgorithm(methodSearchAlgorithm);
         extensionsHandler.setFieldRetriever(fieldRetriever);
 
+        GraphQLAdditionalTypesHandler additionalTypesHandler = new GraphQLAdditionalTypesHandler();
```

Comment:
Maybe create a constructor that gets the parameters below? So we wont need these five lines..